### ISSUE TITLE
Hello World example fails to compile

### ISSUE THREAD

**Issue by Rijenkii** [SergioBenitez/Rocket#671]
Rust version - rustc 1.28.0-nightly (cbc4c8380 2018-06-22)
`rocket` version - 0.3.13
`rocket_codegen` version - 0.3.13

Code:
```rust
#![feature(plugin, decl_macro)]
#![plugin(rocket_codegen)]

#[macro_use] extern crate rocket;

#[get("/<name>/<age>")]
fn hello(name: String, age: u8) -> String {
    format!("Hello, {} year old named {}!", age, name)
}

fn main() {
    rocket::ignite().mount("/hello", routes![hello]).launch();
}
```

Error:
```
   Compiling rocket_codegen v0.3.13
error[E0599]: no associated item named `Lifetime` found for type `syntax::ast::GenericParam` in the current scope
  --> /home/rijenkii/.cargo/registry/src/github.com-1ecc6299db9ec823/rocket_codegen-0.3.13/src/decorators/derive_form.rs:30:25
   |
30 |                         GenericParam::Lifetime(ref def) => Some(def),
   |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ associated item not found in `syntax::ast::GenericParam`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0599`.
error: Could not compile `rocket_codegen`.
```

**Comment by Tiankui** [comment]
same

**Comment by SergioBenitez** [comment]
Fixed in `v0.3.14`, which was just released.
